examples/extended/grails-app/i18n/messages.properties

```diff
@@ -54,3 +54,4 @@ typeMismatch.java.lang.Short=Property {0} must be a valid number
 typeMismatch.java.math.BigDecimal=Property {0} must be a valid number
 typeMismatch.java.math.BigInteger=Property {0} must be a valid number
 typeMismatch=Property {0} is type-mismatched
+securityvalidations.labelMessage.label1=Count to six minus 1
```

Comment:
You may want to add some physical space (new line) between this property and the data binding error properties above to help communicate that they are not related.

Reply:
Agree will fix!